permissions/api_permissions/graphene_permissions.py

```diff
@@ -55,10 +55,11 @@ def has_permission(cls, info: ResolveInfo) -> bool:
     @classmethod
     def has_node_permission(cls, info: ResolveInfo, id: str) -> bool:
         user = info.context.user
-        application = Application.objects.filter(id=id)
+        application = Application.objects.filter(id=id).first()
 
         if application:
-            return user.is_authenticated and can_read_application(user, application)
+            access = user.is_authenticated and can_read_application(user, application)
+            return access
```

Comment:
Nitpick: variable assignment feels a bit unnecessary here since we just return the value

Reply:
oh, this change shouldn't be here. I'll clean it.

Reply:
cleaned 🧹